e2e/staticpvc.go

```diff
@@ -189,6 +189,109 @@ func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock, checkI
 	return err
 }
 
+func validateRBDStaticMigrationPV(f *framework.Framework, appPath string, isBlock, checkImgFeat bool) error {
+	opt := make(map[string]string)
+	var (
+		rbdImageName = "test-static-pv"
+		pvName       = "pv-name"
+		pvcName      = "pvc-name"
+		namespace    = f.UniqueName
+		// minikube creates default class in cluster, we need to set dummy
+		// storageclass on PV and PVC to avoid storageclass name mismatch
+		sc = "storage-class"
+	)
+
+	c := f.ClientSet
+	mon := "rook-ceph-mon-a.rook-ceph.svc.cluster.local:6789"
+
+	size := "4Gi"
+	// create rbd image
+	cmd := fmt.Sprintf(
+		"rbd create %s --size=%d --image-feature=layering %s",
+		rbdImageName,
+		4096,
+		rbdOptions(defaultRBDPool))
+
+	_, e, err := execCommandInToolBoxPod(f, cmd, rookNamespace)
+	if err != nil {
+		return err
+	}
+	if e != "" {
+		return fmt.Errorf("failed to create rbd image %s", e)
+	}
+
+	opt["migration"] = "true"
+	opt["monitors"] = mon
+	if !checkImgFeat {
```

Comment:
This value is always false do we need this one?

Reply:
just kept it for parity with other static test

Reply:
This code will never get hit then why do we need it?